### PR TITLE
Changed cockpit cameras

### PR DESCRIPTION
**Pull request by NoName824**
You can now look left and right with with a flight stick and with ctrl + 1/2/3/4 you can see some parts

ctrl 1
Microsoft Flight Simulator_2021 07 01-14 37_1

ctrl 2
Microsoft Flight Simulator_2021 07 01-14 37_2

ctrl 3
Microsoft Flight Simulator_2021 07 01-14 37_3

ctrl 4
Microsoft Flight Simulator_2021 07 01-14 37_4

### CODE REVIEW

**Comment by FlightSimmer7700** [comment]
Digital Flight Dynamics QA Report
PR Number: #16 
Discord Username: FlightSimmer7700#7700
Date: 01/07/2021

Steps taken to test:
- Spawned in
- Checked all default cameras

Issues (If any):
- On Views "2 and 3" the perspective of the camera is very weird. I would recommend putting that back to normal and moving the camera further back instead. I feel all cameras should keep the same perspective so that the transitions between them look good instead of looking weird as the position and perspective of the camera changes
- I know view 5 isn't included in this pr but I think it should get a quick fix as its currently looking at the ground

Report: Fail
Comments: Views are in the correct positions but the perspective is off and one view not related is broken which could be fixed under the same pr

Media:
image
image
image

**Comment by xZreai** [comment]
Has this been fixed yet?

**Comment by NoName824** [comment]
Not yet I will maybe change it today or tomorrow but I can't put them "back to normal" because then they would look like 5. Also I don't know where I should put the 5th view and what you can see there because technically there are up to 9 vievs I think.

**Comment by NoName824** [comment]
View 2+3 are changed. 5-0 are still missing

**Comment by xZreai** [comment]
I'd make it in this order : 
1 -PFD
2- ND
3- upper ECAM
4- lower -ECAM/MFD
5- throttle
6- RMP panel
7- lower pedestal [flaps/speedbrakes etc]
8- overhead
9- further overhead
0- OIS

**Comment by xZreai** [comment]
Okay we have finalised this order now: 
1- PFD/ND
2- ECAM
3- MFD
4- OIS
5- throttle/RMP panel/engine starter
6- lower pedestal [flaps/speedbrakes etc]
7- overhead
8- further overhead
9- cabin window view 1
0- cabin windows view 2

For now you can omit views 9 and 0 since cabin hasnt been imported to main branch yet, will do once that is done

**Comment by NoName824** [comment]
Microsoft Flight Simulator_2021 07 18-14 51
1
Microsoft Flight Simulator_2021 07 18-14 51_1
2
Microsoft Flight Simulator_2021 07 18-14 51_2
3
Microsoft Flight Simulator_2021 07 18-14 51_3
4
Microsoft Flight Simulator_2021 07 18-14 51_4
5
Microsoft Flight Simulator_2021 07 18-14 51_5
6
Microsoft Flight Simulator_2021 07 18-14 51_6
7

Also I couldn't change 8,9 and 0 yet. I'll look into it after the model is done

**Comment by FlightSimmer7700** [comment]
Digital Flight Dynamics QA Report
PR Number: #16 
Discord Username: FlightSimmer7700#7700
Date: 18/7/21

Steps taken to test:
Loaded in
Checked all camera presets
Checked new cameras

Issues: Cameras don't have the same focal length again so transitions between different cameras look weird. As I mentioned last time can you please have the same or a similar focal length for all views or the same as the default view when loading in so that transitions don't look weird (mostly ctrl + 5 and 6 but all cameras should get the same treatment)

Report: Fail

Comments:
Basically the same issue again

**Comment by NoName824** [comment]
Isn't it the same in the default airliner like the B747? In view one you see the PFD+ND and in the secound FCU + the 5 screens below. That obvisly means that the zoom is diffrent as well or do I understand something wrong.

Microsoft Flight Simulator_2021 07 19-10 07
view 1

Microsoft Flight Simulator_2021 07 19-10 07_1
view 2.

also view 5 and 6 have the excact same zoom and only the Y position is diffrent
